src/java/org/apache/cassandra/utils/memory/MemtablePool.java

```diff
@@ -74,9 +76,19 @@ public void shutdownAndWait(long timeout, TimeUnit unit) throws InterruptedExcep
         ExecutorUtils.shutdownNowAndWait(timeout, unit, cleaner);
     }
 
-
     public abstract MemtableAllocator newAllocator();
 
+    boolean needsCleaning()
+    {
+        return onHeap.needsCleaning() || offHeap.needsCleaning();
+    }
+
+    @Override
+    public String toString()
+    {
+        return String.format("Memory used %.2f/%.2f", onHeap.usedRatio(), offHeap.usedRatio());
```

Comment:
nit: It might be good to label the on-heap and off-heap parts of this for clarity.